tools/execsnoop.py

```diff
@@ -128,8 +130,13 @@
 }
 """
 
+bpf_text.replace("MAXARG", args.max_args)
```

Comment:
You need `bpf_text = bpf_text.replace(...)`.  Otherwise, the statement has no effect and the tool will fail.